config/jobs/ocp-automation/ocp4-upi-powervm/ocp4-upi-powervm-presubmit.yaml

```diff
@@ -1,8 +1,11 @@
 presubmits:
   ocp-power-automation/ocp4-upi-powervm:
     - name: pull-ocp4-upi-powervm-terraform-validate
+      branches:
+        - master
       always_run: true
       decorate: true
+      clone_uri: "git@github.com:ocp-power-automation/ocp4-upi-powervm.git"
```

Comment:
you don't need this because this repo is public

Reply:
oki then also remove it from powervs configs?

Reply:
yes, if that is public :)